components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -127,8 +127,10 @@ func main() {
 	//
 	// Using map is intentional - we ensure that component name is not duplicated.
 	optionalComponentsDisablers := runtime.ComponentsDisablers{
-		"Kiali":  runtime.NewGenericComponentDisabler("kiali", "kyma-system"),
-		"Jaeger": runtime.NewGenericComponentDisabler("jaeger", "kyma-system"),
+		"Kiali":     runtime.NewGenericComponentDisabler("kiali", "kyma-system"),
+		"Jaeger":    runtime.NewGenericComponentDisabler("jaeger", "kyma-system"),
+		"BackupInt": runtime.NewGenericComponentDisabler("backup-init", "kyma-system"),
```

Comment:
please add your components under the `TODO` comment

Reply:
ah I saw your comments too late already hit the merge button. Should I revert it and apply it or should i create a separate PR?